DEV/org.openl.rules/src/org/openl/rules/context/DefaultRulesRuntimeContext.java

```diff
@@ -390,19 +387,19 @@ public void setLocale(java.util.Locale locale) {
 
     static {
         Map<String, Class<?>> contextFields = new TreeMap<>();
-        contextFields.put("currentDate", java.util.Date.class);

```

Comment:
this code is autogenerated from templates like https://github.com/openl-tablets/openl-tablets/blob/master/DEV/org.openl.rules.gen/src/org/openl/codegen/template/DefaultRulesContext-properties.vm

t seems like these templates also must be fixed to remove inserting packages.